ocs_ci/deployment/aws.py

```diff
@@ -177,6 +177,11 @@ def deploy_ocp(self, log_cli_level='DEBUG'):
                 (default: "DEBUG")
         """
         super(AWSIPI, self).deploy_ocp(log_cli_level)
+        if config.DEPLOYMENT['infra_nodes']:
```

Comment:
Post deployment we expect few set of ocs pods to run on the worker,infra tainted nodes. IMO, it will be good to have a verification step for this post OCS deployment.
WDYT?

Reply:
https://github.com/red-hat-storage/ocs-ci/issues/3291

Reply:
created a github issue